test/fail_compilation/jsonBadField1.d

```diff
@@ -0,0 +1,9 @@
+/*
+PERMUTE_ARGS:
+REQUIRED_ARGS: -Xi=UNKNOWN_FIELD_NAME
+TEST_OUTPUT:
+---
+Error: unknown JSON field `-Xi=UNKNOWN_FIELD_NAME`, expected one of compilerInfo, buildInfo, modules, semantics
```

Comment:
Please modify the code to surround each expected option in tick marks, e.g. \`compilerInfo\`, \`buildInfo\`, \`modules\`, \`semantics\`

Reply:
done, thanks for the suggestion